garage/envs/normalized_env.py

```diff
@@ -34,7 +40,14 @@ def __init__(
         self._reward_var = 1.
 
     def _update_obs_estimate(self, obs):
-        flat_obs = flatten(self.env.observation_space, obs)
+        flat_obs = None
+        if isinstance(self.env.observation_space, gym.spaces.Box):
+            flat_obs = np.asarray(obs).flatten()
```

Comment:
ok fine you've convinced me that flat_dim, flatten{_n}, and unflatten{_n} are warranted.

Reply:
I can make them util functions for normalized_env only, as I've removed all other usages of `util.py` in garage.